core/src/main/java/org/mule/runtime/core/api/util/IOUtils.java

```diff
@@ -179,6 +180,18 @@ public static String toString(InputStream input) {
   }
 
   /**
+   * This method wraps {@link org.apache.commons.io.IOUtils}' <code>toString(InputStream, Charset)</code> method but catches any
+   * {@link IOException} and wraps it into a {@link RuntimeException}.
+   */
+  public static String toString(InputStream input, Charset encoding) {
```

Comment:
should we deprecate `toString(CursorStreamProvider cursorStreamProvider)` in favor of this method?